code/modules/antagonists/traitor/syndicate_contract.dm

```diff
@@ -166,8 +166,11 @@
 	if (M.stat != DEAD)
 		// Heal them up - gets them out of crit/soft crit. If omnizine is removed in the future, this needs to be replaced with a
 		// method of healing them, consequence free, to a reasonable amount of health.
-		M.reagents.add_reagent(/datum/reagent/medicine/omnizine, 20)
-
+		//if the species is oozeling change the reagent to regen_ooze since omnizine deals toxin damage to them
+		if(is_species(M, /datum/species/oozeling))
```

Comment:
This should handle all mobs that take inverse toxin damage (Toxin lover trait?)

Reply:
(or you could just give regenerative nanites)

Reply:
I mean yeah and if someone had omnizine on them before they could overdose

Reply:
changed it alltogetherm, is it better now?